### DISCUSSION TITLE
Need help (Could not find artifact)

### DISCUSSION THREAD

**Discussion by italianfelipe** [Minestom/Minestom#1973] [Help] [answered]
Error:
> Could not find artifact com.github.Minestom:Minestom:pom:-SNAPSHOT in jitpack (https://jitpack.io)
> Could not find artifact com.github.Minestom:Minestom:pom:-SNAPSHOT in jitpack (https://jitpack.io)

Maven pom.xml:
`<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>me.felipe</groupId>
    <artifactId>Ferv</artifactId>
    <version>1.0-SNAPSHOT</version>

    <properties>
        <maven.compiler.source>17</maven.compiler.source>
        <maven.compiler.target>17</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    </properties>

    <repositories>
        
        <repository>
            <id>jitpack</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>

    <dependencies>
        
        <dependency>
            <groupId>com.github.Minestom</groupId>
            <artifactId>Minestom</artifactId>
            <version>-SNAPSHOT</version>
            <exclusions>
                <exclusion>
                    <groupId>org.jboss.shrinkwrap.resolver</groupId>
                    <artifactId>shrinkwrap-resolver-depchain</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
    </dependencies>

</project>`

I tried even the id version but they ain't working please help <3

**Comment by vearendev** [comment, marked as answer]
I recommend using [Minestom-CE](https://github.com/hollow-cube/minestom-ce) and Gradle instead

**Reply by italianfelipe** [reply]
is there a way i can use that to make a 1.20.2 server or make it accessible from 1.20.2?

**Reply by vearendev** [reply]
You can wait for the 1.20.2 update, which should be soon, or you can use Via-version on the proxy.
